assets/css/main.scss

```diff
@@ -19,68 +17,5 @@
   {% endif %}
 
   "syntax-highlighting",
-  "typeahead"
```

Comment:
why did you remove this one?

Reply:
That would be an accident, I meant to move it into the `_scss/jekyll-doc-theme.scss` file to keep the main.scss file cleaner.  100% my fault for forgetting about it.